server/api/reviews.js

```diff
@@ -1,14 +1,148 @@
-const router = require('express').Router()
-const {Reviews} = require('../db/models')
-module.exports = router
+const router = require('express').Router();
+const {Reviews} = require('../db/models');
+const {Product} = require('../db/models');
+const {User} = require('../db/models');
+module.exports = router;
 
+// reviews by product
+// reviews by user
+
+// GET all Reviews
 router.get('/', (req, res, next) => {
     Reviews.findAll({
         // explicitly select only the id and email fields - even though
         // users' passwords are encrypted, it won't help if we just
         // send everything to anyone who asks!
-        attributes: ['id', 'email']
+        attributes: ['review', 'rating']
     })
         .then(users => res.json(users))
         .catch(next)
 })
+
```

Comment:
I think these are good but I would put something in the api route between reviews and the id, because if you are trying to get /api/reviews/4 how will it know which id 4 is for. Just a thought